tools/istio-docker.mk

```diff
@@ -127,6 +119,7 @@ docker.proxyv2: pilot/docker/envoy_pilot.yaml.tmpl
 docker.proxyv2: pilot/docker/envoy_policy.yaml.tmpl
 docker.proxyv2: tools/packaging/common/istio-iptables.sh
 docker.proxyv2: pilot/docker/envoy_telemetry.yaml.tmpl
+docker.proxyv2: $(ISTIO_DOCKER)/istio-iptables
```

Comment:
How about `proxy_debug` and `proxytproxy`? You modified the command line in the injection template, so those images will now break.

Reply:
Still missing the dependencies for `proxy_debug` and `proxytproxy`:
```
docker.proxy_debug: $(ISTIO_DOCKER)/istio-iptables
...
docker.proxytproxy: $(ISTIO_DOCKER)/istio-iptables
```